src/pages/ReportDetailsPage.tsx

```diff
@@ -606,7 +607,7 @@ function ReportDetailsPage({policies, report, session, personalDetails}: ReportD
                 <MenuItemWithTopDescription
                     shouldShowRightIcon={!shouldDisableRename}
                     interactive={!shouldDisableRename}
-                    title={reportName}
+                    title={StringUtils.lineBreaksToSpaces(reportName)}
```

Comment:
tbh, I'm not sure if we also need to truncate the title here

Reply:
I have truncated the title there to fix the issue where report details title is displayed on multiple line.
<img width="1710" alt="Screenshot 2024-07-31 at 1 23 07 in the afternoon" src="https://github.com/user-attachments/assets/234018a8-33fc-4acb-a8cf-b1658a6aa695">